### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by mlangbehn**
This change includes a number of updates, including:
  *   Add scripts/resources.sh to simplify other scripts
        *   Update other scripts to use it
    *   Rename scripts/install_bx.sh to scripts/install.sh
        *   Update other files to use new name
    *   Add scripts/quickstart.sh
        *   Update other files to use it
    *   Add test-requirements.txt
        *   Update Travis CI to use it
    *   Add some helpful comments to scripts
    *   Travis CI config
        *   Cache pip
        *   Use fast_finish

Related-Issues: IBM/cloud-journeys#1
Signed-off-by: Matt Langbehn <matthew.langbehn@gmail.com>

### CODE REVIEW

**Review by jkomg — DISMISSED** [review]
LGTM

**Comment by szihai** [comment]
why is there a CI error? Otherwise looks good.

File: tests/test-kubernetes.sh [hunk 1]
```diff
@@ -36,76 +31,44 @@ kubectl_config() {
...
-            fi
-            if [ "$TRIES" -eq 10 ]
-            then
-                echo "Failed finding Wordpress. Error code is $code"
-                exit 1
-            fi
-            TRIES=$((TRIES+1))
-            sleep 5s
-        done
+        if [[ "$(curl -sw '%{http_code}' http://$IP:8080/api/v1/namespaces/default/services/wordpress/proxy/wp-admin/install.php -o /dev/null)" = "200" ]]; then
```
Comment (loafyloaf) [inline tests/test-kubernetes.sh]: travis doesn't like this line:

^-- SC2086: Double quote to prevent globbing and word splitting.

**Review by loafyloaf — CHANGES_REQUESTED** [review]
comment inline

File: tests/test-kubernetes.sh (hunk 1 above)
Comment (mlangbehn) [reply tests/test-kubernetes.sh]: Thanks for pointing that out! I con't believe my linter didn't catch it locally...

**Review by loafyloaf — DISMISSED** [review]

**Review by loafyloaf — COMMENTED** [review]
kubeadm-dind-cluster looks to run successfully, but seems to return 1 anyway...

**Comment by mlangbehn** [comment]
~@loafyloaf, it looks like it's failing to verify the deployment...~

I see what I did...

**Comment by mlangbehn** [comment]
There we go...

**Comment by szihai** [comment]
Look good now!

**Review by loafyloaf — APPROVED** [review]
very nice

**Pull request merged**